From d7dcbf9726607734dffdf01f853b423400f4849d Mon Sep 17 00:00:00 2001
From: Lukas Nykryn <lnykryn@redhat.com>
Date: Wed, 3 Jul 2024 16:57:02 +0200
Subject: [PATCH] netif-naming-scheme: make actually possible to use rhel-9.5
 scheme

In 753e1b1c9b255d528eb8b2a2af072a83eb85d784 we forgot to update the
netif-naming-scheme.c file to allow people to actually use the new
rhel-9.5 scheme.

RHEL-only

Resolves: RHEL-44630
---
 src/shared/netif-naming-scheme.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/shared/netif-naming-scheme.c b/src/shared/netif-naming-scheme.c
index 536ed44b21..9f0c783300 100644
--- a/src/shared/netif-naming-scheme.c
+++ b/src/shared/netif-naming-scheme.c
@@ -44,6 +44,7 @@ static const NamingScheme naming_schemes[] = {
         { "rhel-9.2", NAMING_RHEL_9_2 },
         { "rhel-9.3", NAMING_RHEL_9_3 },
         { "rhel-9.4", NAMING_RHEL_9_4 },
+        { "rhel-9.5", NAMING_RHEL_9_5 },
         /* … add more schemes here, as the logic to name devices is updated … */
 
         EXTRA_NET_NAMING_MAP
